spk/tor/src/service-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Package
+PACKAGE="tor"
+DNAME="Tor"
```

Comment:
The service-part looks good!
Only these first 6 lines are not needed and can be removed!
I didn't compile it yet, but I can start a Travis-build and add it to #3138 if you like?

Reply:
woops. yeah those should be gone.

I

$USER is still the right user? was a little confused with $USER and $EFF_USER \cc @ymartin59